### ISSUE TITLE
Copy qualified name` doesn't work correctly

### ISSUE THREAD

**Issue by mojo2012** [jiangdequan/vscode-java-saber#24]
Hi,

when using the `Copy qualified name` command I always end up having `io.github.jiangdequan` in my clipboard, regardless of where my cursor in a java class is.

It seems that this is used as a default value in case something goes wrong. In that case it would be better to just show a notification.

**Comment by enilu** [comment]
I also encountered the same problem, and i use [VSCode Copy fully qualified name](https://marketplace.visualstudio.com/items?itemName=sandipchitale.vscode-fqn) instead of that。

**Comment by jiangdequan** [comment]
> Hi,
> 
> when using the `Copy qualified name` command I always end up having `io.github.jiangdequan` in my clipboard, regardless of where my cursor in a java class is.
> 
> It seems that this is used as a default value in case something goes wrong. In that case it would be better to just show a notification.

Yes, it will return the default value if the extension can not parse the path correctly.
If you meet the problem again, please show me the full path of the class.

**Comment by mojo2012** [comment]
A better way would be to show an error, instead of misleading the user that everything went fine only to realize on paste that in fact there is a problem.

**Comment by jiangdequan** [comment]
It's a good idea!I will add it in next version.

**Comment by pbodnar** [comment]
Also, unlike the extension linked above, this one just always copies the FQN of the currently opened class - instead of FQN of the thing at cursor.

**Comment by DavidWhom** [comment]
This issue still remains unresolved. 
I also i use [VSCode Copy fully qualified name](https://marketplace.visualstudio.com/items?itemName=sandipchitale.vscode-fqn) instead of that. But it is a command not a right click way to copy.
